### DISCUSSION TITLE
can pixi update itself?

### DISCUSSION THREAD

**Discussion by raceychan** [prefix-dev/pixi#503] [Q&A] [answered]
can pixi update itself?
wasn't able to find it on the docs, am i missing something?

**Comment by wolfv** [comment]
For now you can re-run the installation script. We want to integrate the `self-replace` crate though (for users who didn't install from other package managers like `brew`, `chocolatey`, etc.)

https://github.com/mitsuhiko/self-replace

**Comment by pavelzw** [comment, marked as answer]
There is `pixi self-update` now.

https://pixi.sh/latest/cli/#self-update

If you're using it from a package manager (like `brew`), I would suggest using the package manager to update pixi.
